### PR TITLE
simple JSON db - Submittal

### PR DESCRIPTION
**Pull request by kgentner**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+module.exports = function(grunt) {
+  grunt.loadNpmTasks('grunt-contrib-jshint');
+  grunt.loadNpmTasks('grunt-simple-mocha');
+  grunt.loadNpmTasks('grunt-jscs');
+
+  grunt.initConfig({
+    jshint: {
+      //all: ['lib/*.js'],
```
Comment (toastynerd) [inline Gruntfile.js]: get rid of commented out code.

File: Gruntfile.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      options: {
+        jshintrc: true,
+        node: true
+      },
+      src: ['server.js']
```
Comment (toastynerd) [inline Gruntfile.js]: make sure to run your test code and other support code through at least jshint

File: Gruntfile.js (hunk 1 above)
Comment (kgentner) [inline Gruntfile.js]: Dagnabit! Apologies. Now it's fixed.

File: Gruntfile.js (hunk 2 above)
Comment (kgentner) [inline Gruntfile.js]: Yes, indeed. Now it's fixed.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - "0.10"
+before_script:
+ - mkdir data
```
Comment (toastynerd) [inline .travis.yml:5]: good use of before_script!

File: Gruntfile.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  grunt.loadNpmTasks('grunt-simple-mocha');
+  grunt.loadNpmTasks('grunt-jscs');
+
+  grunt.initConfig({
+    jshint: {
+      options: {
+        jshintrc: true,
+        node: true
+      },
+      src: ['server.js', 'test/**/*.js', 'Gruntfile.js']
```
Comment (toastynerd) [inline Gruntfile.js:14]: nice, way to run the support code through jscs and jshint as well!

File: server.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+var express = require('express');
+var fs = require('fs');
+var app = express();
+
+app.set('port', process.env.PORT || 8080);
+
+app.get('/:username', function(req, res) {
```
Comment (toastynerd) [inline server.js:8]: Nice succinct solution!

File: test/simple_JSON_db_tests.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+var expect = chai.expect;
+
+describe('simple JSON Database', function() {
+  it('should be able to write a single user', function(done) {
+    chai.request('http://localhost:8080')
+    .post('/testname')
+    .end(function(err, res) {
+      expect(err).to.eql(null);
+      expect(res.body.msg).to.eql('New User Saved!');
```
Comment (toastynerd) [inline test/simple_JSON_db_tests.js:17]: Nice, way to test just your code and not the fs library!

**Pull request merged**